### PR TITLE
CJM-57116 : [FCAP ExD] [XDM] Updated offer item schema for index attributes

### PR DESCRIPTION
**Pull request by jshaik-adobe**
JIRA # https://jira.corp.adobe.com/browse/CJM-57116

Added `frequencyCappingConstraintsLastIndex` & `index` attribute to offerItem schema

cc - @mjasrotia  @guayang-adobe

### CODE REVIEW

File: extensions/adobe/experience/decisioning/offerItem.schema.json [hunk 1]
```diff
@@ -27,13 +27,23 @@
...
           "type": "array",
           "items": {
             "type": "object",
             "title": "Frequency capping constraint",
             "description": "Frequency capping constraint defines additional rules based on prior offer proposition, either across all audiences or for a particular profile identity.",
             "properties": {
+              "xdm:index": {
+                "title": "Index",
+                "type": "integer",
+                "description": "Index of the capping object, this is used as suffix for counter id when we have multi capping objects"
```
Comment (usankara) [inline extensions/adobe/experience/decisioning/offerItem.schema.json:45]: let's add hint about index start (0/1)

File: extensions/adobe/experience/decisioning/offerItem.schema.json [hunk 2]
```diff
@@ -27,13 +27,23 @@
             "archived": "Archived"
           }
         },
+        "https://ns.adobe.com/experience/decisioning/offeritem/frequencyCappingConstraintsLastIndex": {
```
Comment (usankara) [inline extensions/adobe/experience/decisioning/offerItem.schema.json:30]: how about using a shorter name fcapArrayLastIndex ..?

Comment (jshaik-adobe) [reply extensions/adobe/experience/decisioning/offerItem.schema.json]: @usankara Updated with shorter name `fCapConstraintsLastIndex`

File: extensions/adobe/experience/decisioning/offerItem.schema.json (hunk 1 above)
Comment (jshaik-adobe) [reply extensions/adobe/experience/decisioning/offerItem.schema.json]: Updated

**Review by mjasrotia — APPROVED** [review]

**Pull request merged**